Firebase/InstanceID/FIRIMessageCode.h

```diff
@@ -65,6 +65,7 @@ typedef NS_ENUM(NSInteger, FIRInstanceIDMessageCode) {
   kFIRInstanceIDMessageCodeService004 = 7004,
   kFIRInstanceIDMessageCodeService005 = 7005,
   kFIRInstanceIDMessageCodeService006 = 7006,
+  kFIRInstanceIDMessageCodeService007 = 7007,
```

Comment:
we start giving the message code a meaning instead of a repeat number. 
So you can do sth like  kFIRIntsanceIDInvalidNetworkSession.